libs-scala/resources-akka/src/main/scala/com/digitalasset/resources/akka/BoundedSourceQueueResourceOwner.scala

```diff
@@ -9,13 +9,16 @@ import com.daml.resources.{AbstractResourceOwner, HasExecutionContext, Releasabl
 
 import scala.concurrent.Future
 
-class BoundedSourceQueueResourceOwner[T, Context: HasExecutionContext](
-    queueGraph: RunnableGraph[BoundedSourceQueue[T]]
+class BoundedSourceQueueResourceOwner[T, E, Context: HasExecutionContext](
```

Comment:
Do we need `E`?

Reply:
Yes, because `BoundedSourceQueue` is invariant in its type argument. We can't use `Any` here.

Reply:
Thanks for 0fe939d